Assets/Scripts/Models/Furniture.cs

```diff
@@ -371,6 +461,11 @@ public Enterability IsEnterable()
         return (Enterability)ret.Number;
     }
 
+    /// <summary>
+    /// Check if the furniture has a unique name and return the name if it has one.
+    /// If the furniture has not unique name, it returns the object type
```

Comment:
it kind of checks if the furniture has a function to determine the sprite name and calls it to get it.